src/sentry/models/dynamicsampling.py

```diff
@@ -274,14 +275,16 @@ def get_project_rules(
             organization=project.organization,
             end_date__gt=now,
             start_date__lt=now,
-        )
+        )[:MAX_CUSTOM_RULES_PER_PROJECT]
```

Comment:
```suggestion
        )[:MAX_CUSTOM_RULES_PER_PROJECT + 1]
        
        if project_rules == MAX_CUSTOM_RULES_PER_PROJECT + 1:
        	metrics.incr("dynamic_sampling.investigation_rules.overflow")
```

Reply:
I'm not really sure we need to do this, it may potentially be useful to know how many orgs go over it but
the limit has to do with Relay's ability to handle big project configurations and therefore is an intrinsic limitation, we can't do much about it.
That being said creating a metric for it is a good idea since it is not hurting anything and may be useful data in the future.

Reply:
I disagree, I understand that this is a limitation of the system but having data to support a possible future decision to figure out if it's worth increasing the capacity of the system is definitely useful.